nowcast/tests/test_data_containers.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+import numpy as np
+
+from delphi_nowcast.data_containers import LocationSeries
+
+
+class TestLocationSeries:
+
+    def test___post_init__(self):
+        with pytest.raises(ValueError) as exc:
+            LocationSeries(dates=[1])
+            assert str(exc.value) == "Must init with both date and values."
```

Comment:
This can be folded into the `pytest.raises()` function as the `match` argument.

Reply:
oh cool didn't know that existed, thanks. tests updated